### ISSUE TITLE
`showTable` and `description` inputs for enhanced accessibility

### ISSUE THREAD

**Issue by vinayakpatil** [swimlane/ngx-charts#1214]
**Is your feature request related to a problem? Please describe.**
Since we don't have keyboard navigation support for most of the charts, it's not accessible.

**Describe the solution you'd like**
Until we add other features around accessibility on the chart component itself, it would be nice to have an export option to show table besides the chart.

**Describe alternatives you've considered**
Highcharts have similar feature: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/export-data/showtable/

**Comment by KingDarBoja** [comment]
Hello @vinayakpatil,

I believe you could implement something similar by making use of `ngx-datatable` package. Unfortunately, to implement some kind of keyboard navigation on the table itself, you have to use the `activate` output as explained on the [docs](https://swimlane.gitbook.io/ngx-datatable/api/table/outputs#activate).

Cheers!
